fixtures/unload.js

```diff
@@ -0,0 +1,26 @@
+import { MongoClient } from 'mongodb';
+
+const connectDb = async (uri) => {
+  const client = await MongoClient.connect(uri, { useNewUrlParser: true });
+  return client;
+}
+
+const unloadFixtures = async (config) => {
+  const client = await connectDb(config.mongo.uri);
+  const db = client.db();
+
+  try {
+    await db.collection('users').deleteMany({ isFixture: true });
```

Comment:
Might be worth considering `Promise.all` to speed up this operation, since order doesn't matter here.